third_party/outline-ss-server/Makefile

```diff
@@ -1,19 +1,19 @@
-VERSION=1.4.0
+VERSION=1.5.0-beta.2
```

Comment:
Ignore this file for now. I'll update this after I cut a new release for `outline-ss-server`.

Reply:
Using the new pre-release 1.5.0.